metron-deployment/packaging/ambari/metron-mpack/pom.xml

```diff
@@ -60,6 +64,19 @@
                             <failIfNoMatch>false</failIfNoMatch>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>metron-short-version</id>
+                        <goals>
+                            <goal>regex-property</goal>
+                        </goals>
+                        <configuration>
+                            <name>metron.short.version</name>
+                            <value>${project.parent.version}</value>
+                            <regex>\.</regex>
```

Comment:
Yeah, SNAPSHOT can be in there, barring some craziness.  It used to have 020BETA in the name.  Just needs to line up with the version, and I'm entirely unsure how Ambari handles it unfortunately.